Formula/czmqpp.rb

```diff
@@ -0,0 +1,81 @@
+class Czmqpp < Formula
+  desc "C++ wrapper for czmq"
+  homepage "https://github.com/zeromq/czmqpp"
+  url "https://github.com/zeromq/czmqpp.git", :tag => "v1.2.0", :revision => "a3c38eb6e3018887a8ecabc77f3029f7edb7b2d0"
+
+  head "https://github.com/zeromq/czmqpp.git"
+
+  option :universal
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "czmq"
```

Comment:
It should work just fine in `def install`. In `test do` the failure is caused by `ENV.cxx` no longer being an just an executable, but an executable with additional arguments. Thus it needs a treatment similar to `ENV.cxxflags`. Something along the lines of:

```ruby
args = ENV.cxx.split + ENV.cxxflags.to_s.split + %W[
  -o test test.cpp
  -I#{include} -L#{lib} -lczmq++
  -L#{Formula["czmq"].opt_lib} -lczmq
]
system(*args)
```

That's not exactly beautiful, but I don't think we have a more elegant solution.